src/overlays/actors/ovl_En_Bji_01/z_en_bji_01.c

```diff
@@ -21,36 +47,421 @@ const ActorInit En_Bji_01_InitVars = {
     (ActorFunc)EnBji01_Update,
     (ActorFunc)EnBji01_Draw,
 };
-*/
 
-#pragma GLOBAL_ASM("./asm/non_matchings/overlays/ovl_En_Bji_01_0x809CCDE0/func_809CCDE0.asm")
+static ColliderCylinderInit sCylinderInit = {
+    {
+        COLTYPE_HIT0,
+        AT_NONE,
+        AC_ON | AC_TYPE_PLAYER,
+        OC1_ON | OC1_TYPE_ALL,
+        OC2_TYPE_1,
+        COLSHAPE_CYLINDER,
+    },
+    {
+        ELEMTYPE_UNK1,
+        { 0x00000000, 0x00, 0x00 },
+        { 0xF7CFFFFF, 0x00, 0x00 },
+        TOUCH_NONE | TOUCH_SFX_NORMAL,
+        BUMP_ON,
+        OCELEM_ON,
+    },
+    { 18, 64, 0, { 0, 0, 0 } },
+};
+
+/* Animations struct */
+static struct_80B8E1A8 D_809CDC7C[4] = {
+    { &D_06000FDC, 1.0f, 0, 0.0f }, /* Looking through telescope */
+    { &D_06005B58, 1.0f, 0, 10.0f }, /* Breathing? Unused? */
+    { &D_06000AB0, 1.0f, 0, 0.0f }, /* Talking */
+    { &D_0600066C, 1.0f, 2, -5.0f }, /* Scratching chin? */
+};
+
+
+void func_809CCDE0(EnBji01* this, GlobalContext* globalCtx) {
+
+    Player* player = PLAYER;
+
+    Vec3f sp58;
+    s32 pad[2];
+
+    Math_Vec3f_Copy(&sp58, &player->actor.world.pos);
```

Comment:
We keep the stack declaration together and with one line of separation from the rest of the code.
```suggestion
void func_809CCDE0(EnBji01* this, GlobalContext* globalCtx) {
    Player* player = PLAYER;
    Vec3f sp58;
    s32 pad[2];

    Math_Vec3f_Copy(&sp58, &player->actor.world.pos);
```